package-lock.json

```diff
@@ -7757,25 +7757,26 @@
       "license": "MIT",
       "dependencies": {
         "@autorest/csharp": "3.0.0-beta.20240625.4",
-        "@typespec/http-client-csharp": "0.1.9-alpha.20250108.2"
+        "@typespec/http-client-csharp": "https://artprodcus3.artifacts.visualstudio.com/A0fb41ef4-5012-48a9-bf39-4ee3de03ee35/29ec6040-b234-4e31-b139-33dc4287b756/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2F6dXJlLXNkay9wcm9qZWN0SWQvMjllYzYwNDAtYjIzNC00ZTMxLWIxMzktMzNkYzQyODdiNzU2L2J1aWxkSWQvNDQ4MjM2Ni9hcnRpZmFjdE5hbWUvYnVpbGRfYXJ0aWZhY3RzX2NzaGFycA2/content?format=file&subPath=%2Fpackages%2Ftypespec-http-client-csharp-0.1.9-alpha.20250115.57.tgz"
       },
       "devDependencies": {
-        "@azure-tools/typespec-autorest": "0.49.0",
-        "@azure-tools/typespec-azure-core": "0.49.0",
-        "@azure-tools/typespec-azure-resource-manager": "0.49.0",
-        "@azure-tools/typespec-azure-rulesets": "0.49.0",
-        "@azure-tools/typespec-client-generator-core": "0.49.1",
+        "@azure-tools/typespec-autorest": "0.50.0",
+        "@azure-tools/typespec-azure-core": "0.50.0",
+        "@azure-tools/typespec-azure-resource-manager": "0.50.0",
+        "@azure-tools/typespec-azure-rulesets": "0.50.0",
+        "@azure-tools/typespec-client-generator-core": "0.50.0",
+        "@azure-tools/typespec-liftr-base": "0.6.0",
```

Comment:
Interesting how we add liftr here.

Reply:
yeah, it is added as a global dependency in the emitter now...